checksum.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Please add the SPDX license header.

Reply:
added in https://github.com/pq-code-package/mlkem-c-aarch64/pull/29/commits/83a92e6dfbe34baee8a6fd25f7e202897ad6d27c